src/components/ha-sidebar.html

```diff
@@ -197,6 +201,10 @@
     this._boundUpdateStyles = this.updateStyles.bind(this);
   },
 
+  tr: function (pg, text, options) {
```

Comment:
btw, I think that when we start converting to Polymer 2 that instead of extending from `Polymer.Element` we extend from `HassBaseClass` and so can inject some stuff we need (like i18n)